make/autoconf/platform.m4

```diff
@@ -114,6 +118,12 @@ AC_DEFUN([PLATFORM_EXTRACT_VARS_FROM_CPU],
       VAR_CPU_BITS=64
       VAR_CPU_ENDIAN=little
       ;;
+    riscv64)
+      VAR_CPU=riscv64
+      VAR_CPU_ARCH=riscv64
+      VAR_CPU_BITS=64
+      VAR_CPU_ENDIAN=little
+      ;;
```

Comment:
Support for `riscv32` was added to the other files, but not here: why?

Reply:
`riscv32 ` was previously leftover option to check the configuration during the cross-compilation.
Given that there is no support on `riscv32 `in our plan, I removed it  just now in other files.